src/main/java/collections/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,27 @@
+package collections.coffee.order;
+
+import java.util.*;
+
+public class CoffeeOrderBoard {
+    TreeMap<Integer, String> orders = new TreeMap<>();
```

Comment:
```suggestion
    private final TreeMap<Integer, String> orders = new TreeMap<>();
```

просто щоб звикати :)